### ISSUE TITLE
Incorrect Python downloaded when installing GPI if Defaults is highest priority channel

### ISSUE THREAD

**Issue by borupdaniel** [conda-forge/gpi-feedstock#1]
Issue:
When attempting to install GPI into a clean environment, conda will download Python 2.7 if Defaults is a higher priority channel than conda-forge. When investigating the package's index.json, it appears that python is not listed as a requirement. This should be python >=3 for GPI to run.
<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
# packages in environment at /Users/m206107/gpi_stack/envs/gpi_condaforge:
#
# Name                    Version                   Build  Channel
```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info
     active environment : gpi_condaforge
    active env location : /Users/m206107/gpi_stack/envs/gpi_condaforge
            shell level : 3
       user config file : /Users/m206107/.condarc
 populated config files : /Users/m206107/.condarc
          conda version : 4.6.14
    conda-build version : not installed
         python version : 3.7.1.final.0
       base environment : /Users/m206107/gpi_stack  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/osx-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://conda.anaconda.org/conda-forge/osx-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /Users/m206107/gpi_stack/pkgs
                          /Users/m206107/.conda/pkgs
       envs directories : /Users/m206107/gpi_stack/envs
                          /Users/m206107/.conda/envs
               platform : osx-64
             user-agent : conda/4.6.14 requests/2.21.0 CPython/3.7.1 Darwin/18.6.0 OSX/10.14.5
                UID:GID : 1851765807:1387956079
             netrc file : None
           offline mode : False
```
</details>

**Comment by borupdaniel** [comment]
Here are the contents of pkgs/gpi-1.0.6-0/info/index.json:

```
{
  "arch": "x86_64",
  "build": "0",
  "build_number": 0,
  "depends": [
    "eigen",
    "fftw",
    "h5py",
    "numpy >=1.11,<2",
    "pillow",
    "psutil",
    "pyopengl",
    "pyqt 4.*",
    "qimage2ndarray",
    "sip",
    "xorg-libsm",
    "xorg-libxrender"
  ],
  "license": "GPL3",
  "license_family": "GPL",
  "name": "gpi",
  "platform": "osx",
  "subdir": "osx-64",
  "timestamp": 1561066235826,
  "version": "1.0.6"
}
```

**Comment by borupdaniel** [comment]
I've tried to address this by including an explicit requirement for python >=3.4 in the run section.

**Comment by borupdaniel** [comment]
Closed by #2
